workflow/scripts/wrapper.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import pandas as pd
+import glob
+import shutil
+import os
+import yaml
+
+with open('config/wrapper_config.yaml', 'r') as f:
+    config = yaml.safe_load(f)
+
+# Get paths from config file
+inpath = config['input_dir']    # Input path containing barcode folders with fastq files
+outpath = config['output_dir']    # Output path for concatenated fastq files
+sample_sheet = pd.read_csv(config['sample_sheet'])    # Sample sheet following Nanopore convention
+
+# Create output directory if it does not exist (but is defined in the config file)
+if not os.path.exists(outpath):
+    os.makedirs(outpath)
+
+# Check if there is a negative control in the sample sheet
+if not sample_sheet['type'].str.contains('neg', na=False).any():
```

Comment:
As above, what if the user specifies this in caps? One way of doing this simple is to use .lower to force both temporarily to be lower case.